python/astro_metadata_translator/translator.py

```diff
@@ -1056,11 +1074,12 @@ def to_property(self):
 # These translator methods can be defined in terms of other properties
 CONCRETE = set()
 
-for name, description in PROPERTIES.items():
+for name, definition in PROPERTIES.items():
```

Comment:
Probably deserves a comment here to say that subclasses with extensions are assumed to not need abstract methods created for them.